### PR TITLE
Resolve issue 14

### LINKED ISSUE #14: Update header with 2024 details & description
image

### LINKED ISSUE #26: Update meta with new date, image & venue
image

### PR DESCRIPTION
**Pull request by jxxcarlson**
This PR resolves #14 & resolves #26.

Please check.

### CODE REVIEW

File: head.html [hunk 1]
```diff
@@ -9,15 +9,15 @@
 <meta property="og:url" content="https://elm.camp/">
 <meta property="og:title" content="Elm Camp">
 <meta property="og:description"
-    content="An Elm unconference taking place in Dallund Castle, Denmark. Wed 28th - Fri 30th June">
+    content="An Elm unconference taking place at Coreyhaines Park . Wed 28th - Fri 30th June">
```
Comment (katjam) [inline head.html:12]:
```suggestion
    content="An Elm unconference taking place at Colehayes Park, Devon UK. Tue 18th - Fri 21st June">
```

File: head.html [hunk 2]
```diff
@@ -9,15 +9,15 @@
...
 <meta property="og:image" content="https://elm.camp/logo.webp">
 
 <!-- Twitter -->
 <meta property="twitter:card" content="summary_large_image">
 <meta property="twitter:url" content="https://elm.camp/">
 <meta property="twitter:title" content="Elm Camp">
 <meta property="twitter:description"
-    content="An Elm unconference taking place in Dallund Castle, Denmark. Wed 28th - Fri 30th June">
+    content="An Elm unconference taking place at Coreyhayes Park, Devon UK. Tue 18th - Fri 21st June">
```
Comment (katjam) [inline head.html:20]:
```suggestion
    content="An Elm unconference taking place at Colehayes Park, Devon UK. Tue 18th - Fri 21st June">
```

**Review by katjam — CHANGES_REQUESTED** [review]
Couple of typos but this looks OK. We could also replace the logo file with the new one from the prospectus.

**Comment by katjam** [comment]
@jxxcarlson I added a png and webp version of the logo because I think maybe the webp version I made does not have transparent background. It's not replacing the old one on my local site, so I pushed up in the interest of time so you could have a play with it.

**Review by jxxcarlson — COMMENTED** [review]
I think it is OK now -- mess with venue spellings and dates fixed

**Comment by jxxcarlson** [comment]
Katja, I merged this PR, please do check the result however.

**Comment by katjam** [comment]
Thanks @jxxcarlson This looks great now! I changed the arrival from Weds --> Tues on `main`

**Pull request merged**